Description: lavu/arm: Check for have_vfp_vm instead of !have_vfpv3 for
 float_dsp_vfp
 .
 This patch disables arm VFP optimizations on processors which do not support
 the deprecated VFP vector extensions. In turn, this fixes the checkasm test
 when run on these processors.
Origin: upstream, https://git.ffmpeg.org/gitweb/ffmpeg.git/commit/f1fd12ef858cf7efe843aa3063894a11b2d1e7d5
Bug: https://trac.ffmpeg.org/ticket/6766
Bug-Debian: https://bugs.debian.org/879800
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/libavutil/arm/float_dsp_init_vfp.c
+++ b/libavutil/arm/float_dsp_init_vfp.c
@@ -36,11 +36,11 @@ void ff_butterflies_float_vfp(float *av_
 
 av_cold void ff_float_dsp_init_vfp(AVFloatDSPContext *fdsp, int cpu_flags)
 {
-    if (!have_vfpv3(cpu_flags)) {
+    if (have_vfp_vm(cpu_flags)) {
         fdsp->vector_fmul = ff_vector_fmul_vfp;
         fdsp->vector_fmul_window = ff_vector_fmul_window_vfp;
     }
     fdsp->vector_fmul_reverse = ff_vector_fmul_reverse_vfp;
-    if (!have_vfpv3(cpu_flags))
+    if (have_vfp_vm(cpu_flags))
         fdsp->butterflies_float = ff_butterflies_float_vfp;
 }
